### PR TITLE
Add note about presentational images

### PR DESCRIPTION
**Pull request by austinstarin**

### CODE REVIEW

File: accessibility.md [hunk 1]
```diff
@@ -42,7 +42,7 @@
   - [4.1](#images--html) Use `<img></img>` for images that are content. Use CSS background images for design "chrome".
 
   <a name="images--alternatives"></a><a name="4.2"></a>
-  - [4.2](#images--alternatives) Use alternative text for any image used as content. Alternative text should provide a description of the image.
+  - [4.2](#images--alternatives) Use alternative text for any image used as content. Alternative text should provide a description of the image. If the image is presentational than favor an empty `alt=""` attribute over `role=presentation`.
```
Comment (bgantick) [inline accessibility.md]: than = then
presentational or decorative

**Comment by austinstarin** [comment]
Thanks for the look. Resolved 282f2c4.

File: accessibility.md [hunk 2]
```diff
@@ -42,7 +42,7 @@
   - [4.1](#images--html) Use `<img></img>` for images that are content. Use CSS background images for design "chrome".
 
   <a name="images--alternatives"></a><a name="4.2"></a>
-  - [4.2](#images--alternatives) Use alternative text for any image used as content. Alternative text should provide a description of the image.
+  - [4.2](#images--alternatives) Use alternative text for any image used as content. Alternative text should provide a description of the image. If the image is presentational or decorative use an empty `alt=""` attribute instead of `role=presentation`.
```
Comment (abea) [inline accessibility.md]: Typo: Last snippet should be `aria-role="presentation"`.

Comment (austinstarin) [inline accessibility.md]: You're suppose to be on vacation! 😄

Comment (abea) [inline accessibility.md]: I'm told I have trouble relaxing.

**Pull request merged**